pymongo/topology_description.py

```diff
@@ -494,7 +494,7 @@ def _updated_topology_description_srv_polling(topology_description, seedlist):
         new_hosts = set(seedlist) - set(sds.keys())
         n_to_add = topology_description.srv_max_hosts - len(sds)
         if n_to_add > 0:
-            seedlist = sample(new_hosts, min(n_to_add, len(new_hosts)))
+            seedlist = sample(sorted(new_hosts), min(n_to_add, len(new_hosts)))
```

Comment:
Why use sorted() instead of list()?

Reply:
It was recommended in the original traceback, I assume in case order was important.

Reply:
Okay I don't think the sort is important but it doesn't hurt either besides a few wasted cycles.